wsgi/odyssy/events/templates/events/event_list.html

```diff
@@ -0,0 +1,19 @@
+<body>
+    {% for event in events %}
+   <li><a href="{% url 'event-view-single' event.id %}">{{ event }}</a></li>
+    {% endfor %}
+
+    {% if events.has_previous %}
+        <a href="?page={{ events.previous_page_number }}&num_items={{num_items}}">Previous</a>
+    {% endif %}
+
+    {% if events.has_next %}
+        <a href="?page={{ events.next_page_number }}&num_items={{num_items}}">Next</a>
+    {% endif %}
+
+    <a href="?num_items=4&page=1">4</a>
+    <a href="?num_items=6&page=1">6</a>
+    <a href="?num_items=8&page=1">8</a>
+    <a href="?num_items=10&page=1">10</a>
+    <a href="?num_items=20&page=1">20</a>
```

Comment:
Number of items per page. You can select dynamically how many items you want per page